src/com/pentalog/pentastagiu/homework2/problema3/Board.java

```diff
@@ -0,0 +1,38 @@
+package com.pentalog.pentastagiu.homework2.problema3;
+
+public class Board {
+    private User[] users;
+    private static int noUsers = 0;
+    private static int chunk = 3;
+
+    public Board() {
+        this.users = new User[chunk];
+    }
+
+    public void RegisterTheUser(User user) {
+        if (user.isRegistering() == true) {
+            if (noUsers % 3 == 0) {
+                User[] new_users = new User[noUsers + chunk];
```

Comment:
fain ca te-ai gandit la mecanismul asta de a crea un array de o anumita lungime si apoi de a il inlocui cu unul mai mare, dar nu la fiecare operatie! ceva asemanator este implementat si in ArrayList